packages/core/test/client.test.ts

```diff
@@ -886,6 +886,7 @@ describe('@bugsnag/core/client', () => {
         featureFlags: [
           { name: 'a', variant: '1' },
           { name: 'b' },
+          // @ts-expect-error
           { name: 'c', variant: 3 }
```

Comment:
`variant` should be a string but we pass a number to check it gets coerced into a string